Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -677,6 +677,14 @@ function_exists('posix_isatty'),
                 'Upgrade your <strong>intl</strong> extension with a newer ICU version (4+).'
             );
 
+            if (class_exists('Symfony\Component\Intl\Intl')) {
+                $this->addRecommendation(
+                    \Symfony\Component\Intl\Intl::getIcuDataVersion() === \Symfony\Component\Intl\Intl::getIcuVersion(),
+                    sprintf('intl ICU version installed on your system (%s) should match the ICU data bundled with Symfony (%s)', \Symfony\Component\Intl\Intl::getIcuVersion(), \Symfony\Component\Intl\Intl::getIcuDataVersion()),
+                    sprintf('Upgrade your <strong>intl</strong> extension with a matching ICU version (%s).', \Symfony\Component\Intl\Intl::getIcuDataVersion())
```

Comment:
Actually, it's not :( you'd need to compile it (icu and php).

Perhaps we should just warn that some data might be inconsistent and needs to be verified for locale used in a project?